code/lib/preview-api/src/modules/preview-web/UrlStore.ts

```diff
@@ -20,19 +20,20 @@ const getQueryString = ({
   extraParams,
 }: {
   selection?: Selection;
-  extraParams?: qs.ParsedQs;
+  extraParams?: Record<PropertyKey, unknown>;
 }) => {
-  const search = typeof document !== 'undefined' ? document.location.search : '';
-  const { path, selectedKind, selectedStory, ...rest } = qs.parse(search, {
-    ignoreQueryPrefix: true,
-  });
-  return qs.stringify(
-    {
+  const search =
+    typeof document !== 'undefined' && document.location.search
+      ? document.location.search.slice(1)
+      : '';
+  const { path, selectedKind, selectedStory, ...rest } = parse(search);
+  return (
+    '?' +
+    stringify({
```

Comment:
Would be nicer to extract the stringify result to a variable, then concatenate using a template string.